concrete/blocks/top_navigation_bar/controller.php

```diff
@@ -192,14 +192,14 @@ protected function getNavigation(): Navigation
     {
         $home = $this->getHomePage();
         $children = $home->getCollectionChildren();
-        $navigation = new Navigation();
+        $navigation = app(Navigation::class);
```

Comment:
I prefer using `$this->app->make(Navigation::class);` when `$this->app` is already available.

Reply:
Good point - I have updated the PR to switch to this format. Thanks for catching this.